core/examples/luigi-example-openui5/webapp/luigi-config.js

```diff
@@ -0,0 +1,60 @@
+//you can now use ES6 goodies here
+Luigi.setConfig({
+  navigation: {
```

Comment:
```suggestion
  navigation: {
    contextSwitcher: false,
```

Reply:
It removes error `customOptionsRenderer` from the console. Also, please take a look at the other logs from the console and try to make them less if it is possible.